agreement_legal/models/agreement.py

```diff
@@ -170,6 +170,20 @@ def _default_stage_id(self):
     company_contact_email = fields.Char(
         related="company_contact_id.email", string="Email"
     )
+    use_parties_content = fields.Boolean(
+        string="Use parties content",
+        default = False,
```

Comment:
```suggestion
        default=False,
```

Reply:
no need to default False, Boolean always False